### ISSUE TITLE
Submodules of website not working

### ISSUE THREAD

**Issue by LoooooKe** [LoooooKe/policr#15]
I had to remove the submodules /website/** because the build failed:
[Travic Job](https://travis-ci.org/LoooooKe/policr/builds/300270270)

You see my changes here:
ae09455cc3d9414ae017bed5b345645134e29623

The page is accessible but obviously not correct rendered:
https://loooooke.github.io/policr/

Please fix the submodule references for the page.

**Comment by rnrbarbosa** [comment]
The site source is on /website, but published on /docs to allow github pages to pick it up there on the master branch as per 
<img width="853" alt="screen shot 2017-11-10 at 20 45 31" src="https://user-images.githubusercontent.com/2978729/32675705-25139840-c658-11e7-9ca5-9aecb18a0df5.png">

**Comment by LoooooKe** [comment]
Yes but the content of website/ should get synchronized by gitmodules which is not correctly configured. The website submodule is not listed in the .gitmodule file..?

**Comment by rnrbarbosa** [comment]
Submodules sorted, not needed anyway. Build was successful. So, static site is published on /docs as reported before.

**Comment by LoooooKe** [comment]
it's working now:
https://loooooke.github.io/policr/

**Comment by LoooooKe** [comment]
Hm looks like the assets still could not be loaded..
